src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig

```diff
@@ -1,13 +1,10 @@
-{% extends "PelagosAppBundle:template:UI.html.twig" %}
-
-{% block head %}
-    {{ parent() }}
-{% endblock %}
+{% extends "PelagosAppBundle::layout.ui.html.twig" %}
 
 {% block body %}
-    <div class="messages error">
+    <div class="messages error alert alert-warning">
```

Comment:
Should this be an error or warning (red)?

Reply:
I mixed it up here and there, we should probably revisit these messages, maybe refactor a thing or two. In this case I felt danger was a bit harsh.